src/main/resources/templates/hint/detail.html

```diff
@@ -43,15 +51,29 @@ <h2><span th:text="${heading}">Nadpis</span></h2>
                 <button type="submit" class="btn btn-primary">Uložit</button>
             </form>
         </div>
-        <div class="col-sm-6">
-            <img th:if="${!#strings.isEmpty(hint.imageUrl)}" class="img-fluid" alt="Obrázek nápovědy" src="" th:src="@{${hint.imageUrl}}" />
-        </div>
         <div class="col-sm-6">
             <img th:if="${!#strings.isEmpty(file)}" class="img-fluid" alt="Obrázek nápovědy" src="" th:src="${file}" />
         </div>
     </div>
 </div>
 
+<script src="//ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
```

Comment:
jQuery by mělo být zahrnuté v externalScripts fragmentu. Takže bych to odstranil a tenhle tvůj script blok bych posunul až za následující div. Aby si jQuery měla dostupné,.